src/procman.cpp

```diff
@@ -347,8 +347,12 @@ procman_data_new (GSettings *settings)
     g_free (color);
 
     /* Sanity checks */
-    swidth = WidthOfScreen (gdk_x11_screen_get_xscreen (gdk_screen_get_default ()));
-    sheight = HeightOfScreen (gdk_x11_screen_get_xscreen (gdk_screen_get_default ()));
+    GdkMonitor *monitor = gdk_display_get_monitor_at_window(gdk_display_get_default(), gtk_widget_get_window(pd->app));
+    GdkRectangle srect;
+    gdk_monitor_get_workarea(monitor, &srect);
+
+    swidth = srect.width;
+    sheight = srect.height;
```

Comment:
Use gtkmm's classes like [Gdk::Monitor](https://developer.gnome.org/gtkmm/stable/classGdk_1_1Monitor.html) for getting the workarea of the monitor in which the application window is located.